packages/grafana-ui/src/components/Modal/getModalStyles.ts

```diff
@@ -28,6 +28,7 @@ export const getModalStyles = stylesFactory((theme: GrafanaTheme2) => {
     `,
     modalBackdrop: css`
       position: fixed;
+      z-index: 1020;
```

Comment:
i think it's fine, just need to get this from the theme instead. something like this?
```suggestion
      z-index: ${theme.zIndex.modalBackdrop};
```

Reply:
Ah nice, yeah, that'll do the trick